x-pack/test/cases_api_integration/security_and_spaces/tests/common/user_actions/get_user_action_stats.ts

```diff
@@ -0,0 +1,206 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import expect from '@kbn/expect';
+import {
+  CaseResponse,
+  CaseSeverity,
+  CaseStatuses,
+  ConnectorTypes,
+} from '@kbn/cases-plugin/common/api';
+
+import {
+  globalRead,
+  obsSec,
+  obsSecRead,
+  noKibanaPrivileges,
+  secOnly,
+  secOnlyRead,
+  superUser,
+  obsOnly,
+  obsOnlyRead,
+} from '../../../../common/lib/authentication/users';
+import { getCaseUserActionStats } from '../../../../common/lib/user_actions';
+import {
+  getPostCaseRequest,
+  persistableStateAttachment,
+  postCaseReq,
+  postCommentActionsReq,
+  postCommentAlertReq,
+  postCommentUserReq,
+  postExternalReferenceESReq,
+} from '../../../../common/lib/mock';
+import { FtrProviderContext } from '../../../../common/ftr_provider_context';
+import {
+  bulkCreateAttachments,
+  createCase,
+  createComment,
+  deleteAllCaseItems,
+  updateCase,
+  superUserSpace1Auth,
+} from '../../../../common/lib/utils';
+
+const getCaseUpdateData = (id: string, version: string) => ({
+  status: CaseStatuses.open,
+  severity: CaseSeverity.MEDIUM,
+  title: 'new title',
+  description: 'new desc',
+  settings: {
+    syncAlerts: false,
+  },
+  tags: ['one', 'two'],
+  connector: {
+    id: 'my-id',
+    name: 'Jira',
+    type: ConnectorTypes.jira as const,
+    fields: null,
+  },
+  id,
+  version,
+});
+
+// eslint-disable-next-line import/no-default-export
+export default ({ getService }: FtrProviderContext): void => {
+  const supertest = getService('supertest');
+  const es = getService('es');
+
+  describe('get_user_action_stats', () => {
+    afterEach(async () => {
+      await deleteAllCaseItems(es);
+    });
+
+    it('returns correct total for comments', async () => {
+      // 1 creation action
+      const theCase = await createCase(supertest, postCaseReq);
+
+      await bulkCreateAttachments({
+        supertest,
+        caseId: theCase.id,
+        params: [
+          // Only this one should show up in total_comments
+          postCommentUserReq,
+          // The ones below count as total_other_actions
+          postExternalReferenceESReq,
+          persistableStateAttachment,
+          postCommentActionsReq,
+          postCommentAlertReq,
+        ],
+      });
+
+      const userActionTotals = await getCaseUserActionStats({ supertest, caseID: theCase.id });
+
+      expect(userActionTotals.total).to.equal(6);
+      expect(userActionTotals.total_comments).to.equal(1);
+      expect(userActionTotals.total_other_actions).to.equal(5);
+      expect(userActionTotals.total).to.equal(
+        userActionTotals.total_comments + userActionTotals.total_other_actions
+      );
+    });
+
+    it('all "other" user action types are included', async () => {
```

Comment:
nit: something like: `it('returns the correct stats when a case update occurs')`